src/controllers/bookmarkController.js

```diff
@@ -0,0 +1,72 @@
+const bookmarkService = require("../services/bookmarkService");
+
+const getAllBookmarks = async (req, res) => {
+  try {
+    const bookmarks = await bookmarkService.getAllBookmarks();
+    res.status(200).json({
+      message: "Bookmarks retrieved successfully",
```

Comment:
true the message here is unnecessary I wanted to have consistency while having message for error and proper response but as you suggested the status code already says it will remove it

Reply:
this is done